torchmetrics/text/wer.py

```diff
@@ -0,0 +1,55 @@
+from typing import Any
+
+from jiwer import compute_measures
+
+from torchmetrics.metric import Metric
+
+
+class WER(Metric):
+    """
```

Comment:
to make sure math gets rendered
```suggestion
    r"""
```

Reply:
Added